---
 configure.ac |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

Index: urfkill-0.6.0~20150318.103828.5539c0d/configure.ac
===================================================================
--- urfkill-0.6.0~20150318.103828.5539c0d.orig/configure.ac
+++ urfkill-0.6.0~20150318.103828.5539c0d/configure.ac
@@ -217,8 +217,7 @@ fi
 # i18n support
 IT_PROG_INTLTOOL([0.40.0])
 
-AM_GNU_GETTEXT_VERSION([0.17])
-AM_GNU_GETTEXT([external])
+AM_GLIB_GNU_GETTEXT
 
 AC_SUBST([GETTEXT_PACKAGE],[${PACKAGE_TARNAME}])
 AC_DEFINE([GETTEXT_PACKAGE],[PACKAGE_TARNAME],[Define to the gettext package name])
