---
 configure.ac |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

Index: b/configure.ac
===================================================================
--- a/configure.ac
+++ b/configure.ac
@@ -211,8 +211,7 @@ fi
 # i18n support
 IT_PROG_INTLTOOL([0.40.0])
 
-AM_GNU_GETTEXT_VERSION([0.17])
-AM_GNU_GETTEXT([external])
+AM_GLIB_GNU_GETTEXT
 
 AC_SUBST([GETTEXT_PACKAGE],[${PACKAGE_TARNAME}])
 AC_DEFINE([GETTEXT_PACKAGE],[PACKAGE_TARNAME],[Define to the gettext package name])
